cfgov/wellbeing/jinja2/wellbeing/home.html

```diff
@@ -850,82 +834,86 @@ <h3 class="a-legend">
-                                       data-gtm-category="Financial Well-Being Tool Interaction">
-                                <label class="a-label" for="age-62-plus">
-                                    62+
-                                </label>
-                            </div>
-                        </li>
-                    </ul>
-                </fieldset>
-                <fieldset class="o-form_fieldset" id="method">
-                    <h3 class="a-legend">
-                        Select how you completed the questionnaire. This changes the scoring calculation.
-                    </h3>
-                    <ul class="m-list m-list__unstyled content-l">
-                        <li class="content-l_col content-l_col-1-2">
-                            <div class="m-form-field m-form-field__radio m-form-field__lg-target">
-                                <input class="a-radio"
-                                       type="radio"
-                                       name="method"
-                                       id="read-self"
-                                       value="read-self"
-                                       data-gtm-action="Questionnaire Radio Button Clicked"
-                                       data-gtm-label="Select how you completed the questionnaire: I read and answered the questions myself"
-                                       data-gtm-category="Financial Well-Being Tool Interaction"
-                                       checked>
-                                <label class="a-label" for="read-self">
-                                    I read and answered the questions myself
-                                </label>
-                            </div>
-                        </li>
-                        <li class="content-l_col content-l_col-1-2">
-                            <div class="m-form-field m-form-field__radio m-form-field__lg-target">
-                                <input class="a-radio"
-                                       type="radio"
-                                       name="method"
-                                       id="read-to-me"
-                                       value="read-to-me"
-                                       data-gtm-action="Questionnaire Radio Button Clicked"
-                                       data-gtm-label="Select how you completed the questionnaire: I read the questions to someone else and recorded their answers"
-                                       data-gtm-category="Financial Well-Being Tool Interaction">
-                                <label class="a-label" for="read-to-me">
-                                    I read the questions to someone else and recorded their answers
-                                </label>
-                            </div>
-                        </li>
-                    </ul>
-                </fieldset>
+            <div class="o-well">
+                <div class="o-form_group">
```

Comment:
I am not enthused about adding another wrapper here just to get some margin. Could we add this bottom margin to all `o-form_fieldsets`, or are there places where we don't want it? Failing that, could we just use a margin utility class on the fieldset?

Reply:
There are places we don't want it and the groups have been the default layout elsewhere. I'd rather stick to consistency and tackle that in a new issue.

Reply:
Fine for now, but I really want to kill `o-form_group`. It really shouldn't be necessary.

Reply:
@Scotchester PRs always accepted 😛